Project.toml

```diff
@@ -0,0 +1,20 @@
+name = "SkyCoords"
+uuid = "fc659fc5-75a3-5475-a2ea-3da92c065361"
+authors = "Kyle Barbary, Mosé Giordano, and contributors"
+version = "0.4.0"
+
+[deps]
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
+
+[compat]
+StaticArrays = "≥ 0.8.0"
```

Comment:
We need upper bounds on the dependencies versions in order to be able to have automerging in the general registry: https://discourse.julialang.org/t/announcement-automatic-merging-for-the-general-registry/29961